packages/mui-utils/src/deepmerge.ts

```diff
@@ -6,6 +6,20 @@ export interface DeepmergeOptions {
   clone?: boolean;
 }
 
+function clonedeep<T>(source: T): T | Record<keyof any, unknown> {
```

Comment:
Could you call this function `deepClone`? I know that `deepmerge` doesn't follow conventions (should be `deepMerge`), but let's ensure new functions are named consistently with most of the code.

Reply:
@michaldudak sure